tests/pages/contribute/stories.py

```diff
@@ -0,0 +1,36 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+
+from .base import ContributeBasePage
+
+
+class ContributeStoriesPage(ContributeBasePage):
+
+    _url = '{base_url}/{locale}/contribute/stories/shreyas'
```

Comment:
https://www.mozilla.org/en-US/contribute/stories/ruben/ doesn't have the toggle - is that a bug?